src/impl/vamp/bindings/common.hh

```diff
@@ -424,13 +473,20 @@ namespace vamp::binding
             .def_ro(
                 "iterations", &RH::Roadmap::iterations, "Number of iterations taken to construct roadmap.");
 
+        submodule.def("halton", RH::halton, "Creates a new Halton sampler.");
+
+#if defined(__x86_64__)
+        submodule.def("xorshift", RH::xorshift, "Creates a new XORShift sampler.");
+#endif
+
         submodule.def(
             "rrtc",
             RH::rrtc_single,
             "start"_a,
             "goal"_a,
             "environment"_a,
-            "settings"_a = vamp::planning::RRTCSettings(),
+            "settings"_a,
+            "rng"_a,
```

Comment:
Is it worth providing a default value for the RNG, as well as keeping the default value for the settings? Also on all the other methods that now take a RNG argument.

Reply:
I actually think we should make it so the user has to specify the settings for the planner to avoid footguns with default settings. We can put default setting logic in the Python API.

Reply:
From discussion: OK, but we need to rework the Python API and keep in mind how to ensure that defaults remain consistent between Python and C++.